apps-rendering/src/components/Headline/GalleryHeadline.tsx

```diff
@@ -0,0 +1,69 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	from,
+	headline,
+	neutral,
+	remSpace,
+} from '@guardian/source-foundations';
+import { grid } from 'grid/grid';
+
+// ----- Component ----- //
+
+const styles = css`
+	${headline.medium({ fontWeight: 'bold' })}
+	background-color: ${neutral[7]};
+	color: ${neutral[100]};
+	padding: 0 ${remSpace[5]} ${remSpace[9]} 0;
+	${grid.between('centre-column-start', 'centre-column-end')}
+	grid-row: 3 / 5;
+
+	${from.tablet} {
+		${grid.span('centre-column-start', 12)}
+	}
+
+	${from.desktop} {
+		${headline.xlarge({ fontWeight: 'bold' })}
+		${grid.span('centre-column-start', 8)}
+	}
+`;
+
+const backgroundStyles = css`
+	background-color: ${neutral[7]};
+	${grid.between('viewport-start', 'centre-column-end')}
+	grid-row: 3 / 5;
+
+	${from.mobileLandscape} {
+		${grid.between('viewport-start', 'viewport-end')}
```

Comment:
This could be `grid.column.all` I think?

```suggestion
		${grid.column.all}
```